x/concentrated-liquidity/types/msgs.go

```diff
@@ -0,0 +1,106 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+// constants.
+const (
+	TypeMsgCreatePosition   = "create-position"
+	TypeMsgWithdrawPosition = "withdraw-position"
+)
+
+var _ sdk.Msg = &MsgCreatePosition{}
+
+func (msg MsgCreatePosition) Route() string { return RouterKey }
+func (msg MsgCreatePosition) Type() string  { return TypeMsgCreatePosition }
+func (msg MsgCreatePosition) ValidateBasic() error {
+	_, err := sdk.AccAddressFromBech32(msg.Sender)
+	if err != nil {
+		return sdkerrors.Wrapf(sdkerrors.ErrInvalidAddress, "Invalid sender address (%s)", err)
```

Comment:
Please remove `sdkerrorss` wraps if possible. Unfortunately, these might lead to state breaks if changed between patches. As a result, we are trying to move away from using them.